decent-dao/fractal-interface
# Token data

## Description

This PR:
1) Adds module contract objects to the modules array
2) Adds governance token data to the Gnosis Governance data. This also includes the token voting contract object and the Usul voting strategy object
3) Adds a temporary delegate page & route for testing purposes
4) Fixes bug where the predicted token address wasn't correct

## Notes



This adds a temporary route to the delegate page at /<dao address>/delegate
This route is for testing purposes so that we can delegate votes to enable votes casting and such.
This delegate component will likely be moved into a modal in a future PR, and won't actually have its own route
This PR does not include styling yet, this can be added in a future PR

## Issue / Notion doc (if applicable)


## Testing
1) Create a Usul DAO, and allocate some of the governance tokens to your address
2) Once on the DAO home page, add `/delegate` to the end of the URL route
3) Try allocating tokens to your address and other addresses, and verify that the `delegatee` and `voting weight` update accordingly

## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/84364476/200350342-f37ede67-4f7e-4fc8-a3ee-7c20f514c8ed.png)
